src/compiler/crystal/command/format.cr

```diff
@@ -72,29 +72,19 @@ class Crystal::Command
     )
       @format_stdin = files.size == 1 && files[0] == "-"
 
-      includes = normalize_paths includes
-      excludes = normalize_paths excludes
+      includes = includes.map { |p| Crystal.normalize_path p }
+      excludes = excludes.map { |p| Crystal.normalize_path p }
       excludes = excludes - includes
       if files.empty?
         files = Dir["./**/*.cr"]
       else
-        files = normalize_paths files
+        files = files.map { |p| Crystal.normalize_path p }
```

Comment:
Why not move `#map { |path| Crystal.normalize_path path }` into the `normalize_paths` helper instead? :)

Reply:
'cause the atomic version was needed in the other command and doing another overload for a map was not worth it IMO.